Squablr/View Controllers/ProfileViewController.m

```diff
@@ -60,21 +137,53 @@ -(void) queryUserProfileInfo {
     }];
 }
 
+- (void)queryUserImages {
+    PFQuery *profileQuery = [PFQuery queryWithClassName:@"Profile"];
+    [profileQuery whereKey:@"name" equalTo:[PFUser currentUser].username];
+    // fetch data asynchronously
+    [profileQuery findObjectsInBackgroundWithBlock:^(NSArray *profiles, NSError * _Nullable error) {
+        if (profiles) {
```

Comment:
Why are we querying here? Seems like you aren't really doing anything with the profiles that are returned

Reply:
That's true, will fix this.